polymesh_schema.json

```diff
@@ -431,8 +431,7 @@
         "Custom": "Vec<u8>"
       }
     },
-    "InvestorZKProofData": "Lookup336",
-    "Lookup336": {
+    "InvestorZKProofData": {
```

Comment:
FYI, the use of `Lookup336` here was a hack because of a bug in `@polkadot/api: 7.6.1`.